src/popover/index.js

```diff
@@ -85,6 +87,9 @@ export default createComponent({
           },
         ],
       });
+      window.addEventListener('animationend', popper.update);
+      window.addEventListener('transitionend', popper.update);
```

Comment:
If the popper is not showed or deactivated when inside a `<keep-alive>`, should we calling `popper.update()` ?